CompatBot/Utils/ResultFormatters/LogParserResultFormatter.GeneralNotesSection.cs

```diff
@@ -265,6 +265,10 @@ private static async Task BuildNotesSectionAsync(DiscordEmbedBuilder builder, Lo
                     {
                         if (threadCount < 12)
                             notes.Add("⚠ Six cores or more is recommended for Ryzen CPUs");
+                        if (cpu.EndsWith('U')
+                            || cpu.EndsWith('H')
+                            || cpu.EndsWith("HS"))
```

Comment:
idk if h and hs should be here, similarly to intel

Reply:
What do you mean? Checked all Ryzen SKUs and they only refer to mobile parts. Do you mean that they might ve performant enough?

Reply:
When results with Zen 2 based laptops actually arrive, I'll try specifying the filters further. Until then I'd say filtering for H(S) suffices.